Sources/JavaKit/JavaKitVM/JavaVirtualMachine.swift

```diff
@@ -54,6 +56,14 @@ public final class JavaVirtualMachine: @unchecked Sendable {
     // Construct the complete list of VM options.
     var allVMOptions: [String] = []
     if !classPath.isEmpty {
+      for path in classPath {
+        do {
+          let fd = try FileDescriptor.open(path, .readOnly)
+          try? fd.close()
```

Comment:
Can't we do this without a new dependency? I don't love FileManager but it does have such API: `FileManager.default.fileExists(atPath: path)` so we con't have to include another dependency (yet).

https://developer.apple.com/documentation/foundation/filemanager/1410277-fileexists